server/sonar-web/src/main/js/apps/overview/meta/Meta.tsx

```diff
@@ -83,27 +77,28 @@ export default class Meta extends React.PureComponent<Props> {
           qualifier={component.qualifier}
         />
 
-        {shouldShowQualityGate && (
-          <MetaQualityGate
-            gate={qualityGate}
-            organization={hasOrganization && component.organization}
-          />
-        )}
-
-        {shouldShowQualityProfiles && (
-          <MetaQualityProfiles
```

Comment:
According to the mock up from Laura `MetaQualityGate` and `MetaQualityProfiles` should be grouped in the same `overview-meta-card`